src/aequitas/plotting.py

```diff
@@ -0,0 +1,1064 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+import matplotlib.gridspec as gridspec
+import re
+import seaborn as sns
+import math
+from aequitas.squarify_flipped import normalize_sizes, squarify, padded_squarify, \
+    squarify_plot_rects
+import warnings
+
+# Authors: Pedro Saleiro <saleiro@uchicago.edu>
+#          Loren Hinkson
+#
+# License: Copyright \xa9 2018. The University of Chicago. All Rights Reserved.
+
+class Plotting(object):
+    """
+    """
+
+    def __init__(self, key_metrics=None, key_disparities=None):
+        """
+        :param key_metrics: Set default absolute group metrics for all subplots
+        :param key_metrics: Set default disparity metrics for all subplots
+        :param fill_divbyzero:
+        """
+        if not key_metrics:
+            self.key_metrics = ['pprev', 'ppr', 'fdr', 'for', 'fpr', 'fnr']
+        else:
+            self.key_metrics = key_metrics
+        if not key_disparities:
+            self.key_disparities = \
+                ['pprev_disparity', 'ppr_disparity', 'fdr_disparity',
+                     'for_disparity', 'fpr_disparity', 'fnr_disparity']
+        else:
+            self.key_disparities = key_disparities
+
+        # Define mapping for condiitonal coloring based on fairness
+        # determinations
+        self.__metric_parity_mapping = {
+                'ppr_disparity': 'Statistical Parity',
+                'pprev_disparity': 'Impact Parity',
+                'precision_disparity': 'Precision Parity',
+                'fdr_disparity': 'FDR Parity',
+                'for_disparity': 'FOR Parity',
+                'fpr_disparity': 'FPR Parity',
+                'fnr_disparity': 'FNR Parity',
+                'tpr_disparity': 'TPR Parity',
+                'tnr_disparity': 'TNR Parity',
+                'npv_disparity': 'NPV Parity',
+                'ppr': 'Statistical Parity',
+                'pprev': 'Impact Parity',
+                'precision': 'Precision Parity',
+                'fdr': 'FDR Parity',
+                'for': 'FOR Parity',
+                'fpr': 'FPR Parity',
+                'fnr': 'FNR Parity',
+                'tpr': 'TPR Parity',
+                'tnr': 'TNR Parity',
+                'npv': 'NPV Parity'
+                }
+
+
+    @property
+    def metric_parity_mapping(self):
+        ''' Getter for metric_parity_mapping '''
+        return self.__metric_parity_mapping
+
+
+    def __nearest_quartile(self, x):
+        rounded = round(x * 4) / 4
+        if rounded > x:
+            return rounded
+        else:
+            return rounded + 1 / 4
```

Comment:
methods like this would _usually_ be indicated as "internal / helper" by simply adding a _single_ leading underscore -- not via the pseudo-private language feature, (which mucks with inheritance but which is otherwise much the same).

also, this is really a static method. you _could_ simply define it as a function at the module level, (not as a method). or, if you prefer to keep it under the `Plotting` namespace and/or make it available for inheritance, of course it can be a (static) method.

that is, for example, this will function the same as what you have:

    @staticmethod
    def _nearest_quartile(number):
        ...